src/Symfony/Component/Notifier/Message/SmsMessage.php

```diff
@@ -23,15 +23,17 @@ final class SmsMessage implements MessageInterface
     private $transport;
     private $subject;
     private $phone;
+    private $options;
 
-    public function __construct(string $phone, string $subject)
+    public function __construct(string $phone, string $subject, ?MessageOptionsInterface $options = null)
```

Comment:
```suggestion
    public function __construct(string $phone, string $subject, MessageOptionsInterface $options = null)
```
To stay consistent with the `ChatMessage`

Reply:
Done